entity-framework/efcore-and-ef6/porting/port-detailed-cases.md

```diff
@@ -66,7 +66,7 @@ EF6 supported table-per-hierarchy (TPH), table-per-type (TPT) and table-per-conc
 
 ### Attributes
```

Comment:
I would explicitly mention in the mapping section that inherited types of a mapped entity type are not discovered automatically by EF Core. This has tripped up many people in the past.